dashboard/app/views/pd/workshop_enrollment/attended.html.haml

```diff
@@ -0,0 +1,5 @@
+%p
+  You have already attended this workshop.
+  %p
```

Comment:
Nested `<p>` tags intentional?

Reply:
nope. Will remove

Reply:
Copied from [here](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/pd/workshop_enrollment/closed.html.haml). I'll fix that one too